src/configs/ToastConfig.js

```diff
@@ -33,7 +32,7 @@ const styles = StyleSheet.create({
 
 const styleAsNative = StyleSheet.create({
   container: {
-    backgroundColor: COLORS.gray5,
+    backgroundColor: '#4D4D4D',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Hardcoded color values can lead to inconsistency and difficulty in maintaining the code. Consider using a centralized color configuration.
```diff
-    backgroundColor: '#4D4D4D',
+    backgroundColor: colors.gray5,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->